src/console_cmds.cpp

```diff
@@ -1154,6 +1168,37 @@ DEF_CONSOLE_CMD(ConExec)
 	return true;
 }
 
+DEF_CONSOLE_CMD(ConSchedule)
+{
+	if (argc < 3 || std::string_view(argv[2]) != "on-next-month") {
+		IConsolePrint(CC_HELP, "Schedule a local script to execute later. Usage: 'schedule <script> on-next-month'.");
+		return true;
+	}
+
+	/* Check if the file exists. It might still go away later, but helpful to show an error now */
+	FILE *script_file = FioFOpenFile(argv[1], "r", BASE_DIR);
+	if (script_file == nullptr) {
+		IConsolePrint(CC_ERROR, "Script file '{}' not found.", argv[1]);
+		return true;
+	}
+	FioFCloseFile(script_file);
+
+	/* We only support a single script scheduled, so we tell the user what's happening if there was already one */
+	const std::string filename = std::string(argv[1]);
+	if (_scheduled_filename != "" and filename == _scheduled_filename) {
```

Comment:
Avoid `""` again. C++ uses `&&`, not `and`.

Reply:
Ah I was in Python mode!